build.ps1

```diff
@@ -5,16 +5,26 @@
     [Parameter(Position = 1)]
     [string] $Version = "73.1.12.0",
     [Parameter(Position = 2)]
-    [string] $AssemblyVersion = "73.1.12.0"
+    [string] $AssemblyVersion = "73.1.12.0",
+    [string] $RedistVersion,
+    [string] $Suffix
+
 )
 
 $WorkingDir = split-path -parent $MyInvocation.MyCommand.Definition
 $CefSln = Join-Path $WorkingDir 'CefSharp3.sln'
 
 # Extract the current CEF Redist version from the CefSharp.Core\packages.config file
 # Save having to update this file manually Example 3.2704.1418
-$CefSharpCorePackagesXml = [xml](Get-Content (Join-Path $WorkingDir 'CefSharp.Core\Packages.config'))
-$RedistVersion = $CefSharpCorePackagesXml.SelectSingleNode("//packages/package[@id='cef.sdk']/@version").value
+$PackagesXmlFilename = Join-Path $WorkingDir 'CefSharp.Core\Packages.config'
+$CefSharpCorePackagesXml = [xml](Get-Content ($PackagesXmlFilename))
+$RedistVersionNode = $CefSharpCorePackagesXml.SelectSingleNode("//packages/package[@id='cef.sdk']/@version")
```

Comment:
Changing package versions shouldn't happen as part of the build, it should be done prior to executing the script. This also only updates the `CefSharp.Core` package, doesn't take into account `CefSharp.BrowserSubprocess.Core`, so this could easily lead to a broken build.